torch/csrc/utils/python_arg_parser.h

```diff
@@ -74,6 +74,8 @@
 #include <string>
 #include <vector>
 
+#define TH_MAX_ARGS 32
```

Comment:
This macro is only used in the cpp file, can we define it there? Also, there's no need for it to be a macro; constexpr int will work just fine.

Reply:
What will happen if we try to define a function with more than the max number of arguments?

Reply:
Yeah. I will convert it to constexp.  

When number of arguments is more than 32, it will lead to an error. I chose 32 because Numpy does it.

Reply:
Bumping this comment